### ISSUE TITLE
Compression GZIP when uploading files to Athena/S3

### ISSUE THREAD

**Issue by DyfanJones** [DyfanJones/RAthena#18]
Support GZIP compression:

https://docs.aws.amazon.com/athena/latest/ug/compression-formats.html

>Use the GZIP compression in Athena for querying Amazon Kinesis Data Firehose logs. Athena and Amazon Kinesis Data Firehose each support different versions of SNAPPY, so GZIP is the only compatible format.

**Comment by DyfanJones** [comment]
gzip is not supported in dev version: `1.3.0.9000`

**Comment by OssiLehtinen** [comment]
I see you have closed this issue, but would like to point out, that having GZIPped csv's for Athena to read is not optimal, as GZIP cannot be run in parallel, meaning Athena can assign only one worker for reading per file, which occasionally leads to surprisingly long read times when running queries.

**Comment by DyfanJones** [comment]
RAthena only supports this functionality and is not set as default. If you know a better compression method for csv's that works with Athena please let me know. 

RAthena also supports snappy compression for parquet files through the [`arrow`](https://github.com/apache/arrow/tree/master/r) package. I believe Athena can work with this better.

**Comment by DyfanJones** [comment]
@OssiLehtinen opened up another ticket to address different compression algorithms: https://github.com/DyfanJones/RAthena/issues/36 when uploading flat files to AWS Athena
